packages/devtools/test/support/flutter_test_environment.dart

```diff
@@ -42,6 +42,7 @@ class FlutterTestEnvironment {
   set beforeTearDown(VoidAsyncFunction f) => _beforeTearDown = f;
 
   bool _needsSetup = true;
+  bool get _needsTeardown => !_needsSetup;
```

Comment:
nit: remove this flag. I think checking _needsSetup directly is fine and a little clearer

Reply:
This was an overlap with https://github.com/flutter/devtools/pull/214. I'll rebase on top of that once it's merged and ensure this is gone. Thanks!